examples/kubernetes/vtctld-service.yaml

```diff
@@ -14,4 +14,4 @@ spec:
   selector:
     component: vtctld
     app: vitess
-
+  type: LoadBalancer
```

Comment:
I think we should avoid creating a load balancer when we don't need it, since they have an [hourly service charge](https://cloud.google.com/compute/docs/load-balancing/). I guess that means we need to turn this into a template. You can set "type: ClusterIP" in the default case.